### PR TITLE
Mobile Field (backend) and Team Sorting (frontend)

### PR DESCRIPTION
**Pull request by apanjain**
Changed mobile fields from integer to string as they were not fitting in Postgres constraints(limited size of INT type).
Also, modify the Team Card to show prioritized contact information. And display the team in a sorted manner (yearwise, then alphabetically)

### CODE REVIEW

File: frontend/src/components/TeamCard/index.js [hunk 1]
```diff
@@ -22,11 +23,25 @@ function WrappedComponent(props) {
             <img className={styles.commonIcon} src={fb} alt={"facebook"} />
           </div>
         </a>
-        <a href={"tel:" + props.member.contact}>
-          <div className={styles.call}>
-            <img className={styles.commonIcon} src={call} alt={"phone"} />
-          </div>
-        </a>
+        {props.member.linkedin !== undefined &&
```
Comment (hacksdump) [inline frontend/src/components/TeamCard/index.js:26]: Disable or remove the icons if we don't have facebook or mobile number in backend.
I guess disabled should be okay to maintain a consistent look with other cards that might have the data.

File: backend/pages/models.py [hunk 2]
```diff
@@ -152,8 +152,11 @@ class StudentTeam(models.Model):
         null=True,
         choices=year_choices_team
     )
-    mobile = models.IntegerField(
+    mobile = models.CharField(
```
Comment (hacksdump) [inline backend/pages/models.py:155]: This is fine for this PR. But this needs to be refactored as described in #101 to reduce redundancy.
Do that in another PR.

**Review by hacksdump — CHANGES_REQUESTED** [review]

File: frontend/src/components/TeamCard/index.js (hunk 1 above)
Comment (apanjain) [reply frontend/src/components/TeamCard/index.js]: Done
